index.bs

```diff
@@ -801,6 +837,9 @@ spec:css-syntax-3;
             4.  |options|.{{CredentialRequestOptions/mediation}} is not
                 "{{CredentialMediationRequirement/required}}".
 
+            5.  |options|.{{CredentialRequestOptions/mediation}} is not
```

Comment:
The surrounding algorithm should match https://webidl.spec.whatwg.org/#es-enumeration (throw a `TypeError`) if the caller passes "conditional" to a credential type that would return `false` from `isConditionalMediationAvailable()`.

Reply:
Alright, I added a check for every credential interface that is an input to this algorithm. PTAL.